docs/csharp/properties.md

```diff
@@ -125,7 +125,7 @@ a read-write property with no validation. By writing the code you want in the
 
 You can write code in the `set` accessor to ensure that the values represented
 by a property are always valid. For example, suppose one rule for the `Person`
-class is that the name cannot be blank, or whitespace. You would write that as
+class is that the name cannot be blank, or white space. You would write that as
```

Comment:
Unrelated to your PR: Could you delete the comma between "blank" and "or" both here and in line 149?

Reply:
Done!